src/Eccube/Application.php

```diff
@@ -618,11 +622,11 @@ public function loadPlugin()
 
             // Type: Event
             if (isset($config['event'])) {
-                $class = '\\Plugin\\' . $config['code'] . '\\' . $config['event'];
+                $class = '\\Plugin\\'.$config['code'].'\\'.$config['event'];
                 $subscriber = new $class($this);
 
-                if (file_exists($dir->getRealPath() . '/event.yml')) {
-                    foreach (Yaml::Parse($dir->getRealPath() . '/event.yml') as $event => $handlers) {
+                if (file_exists($dir->getRealPath().'/event.yml')) {
+                    foreach (Yaml::Parse($dir->getRealPath().'/event.yml') as $event => $handlers) {
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'loadPlugin'.